changelog/Microsoft.IdentityProtectionServices.json

```
@@ -1,5 +1,31 @@
 {
 	"changelog": [
+		{
+			"ChangeList": [
+				{
+					"Id": "8aa81736-9387-4a42-ab7a-e69eaf53fb8c",
+					"ApiChange": "Property",
+					"ChangedApiName": "clientApplications",
+					"ChangeType": "Addition",
+					"Description": "Added the **clientApplications** property to the [conditionalAccessConditionSet](https://docs.microsoft.com/en-us/graph/api/resources/conditionalAccessConditionSet?view=graph-rest-beta) resource. Use this property to configure the service principals and workload identities to include in or exclude from a conditional access policy.",
+					"Target": "conditionalAccessConditionSet"
+				},
+				{
+					"Id": "7ab08cd6-b443-4855-b625-9b80b1b92ea3",
```

Use the same ID for all ChangeList objects in your changelog.
```suggestion
					"Id": "8aa81736-9387-4a42-ab7a-e69eaf53fb8c",
```